src/pages/RegisterForMeeting.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { Center, Text, VStack } from '@chakra-ui/react';
+import { useHistory, useParams } from 'react-router';
+import Loading from '../components/common/Loading';
+import { useRegisterAsParticipantMutation } from '../__generated__/graphql-types';
+
+const RegisterForMeeting: React.FC = () => {
+  const history = useHistory();
+  const [registerAsParticipant, { data, loading, error }] = useRegisterAsParticipantMutation();
+
+  const { meetingId } = useParams<{ meetingId: string }>();
+
+  useEffect(() => {
+    if (meetingId && !loading && !data && !error) {
+      registerAsParticipant({ variables: { meetingId } });
+    }
+  }, [meetingId, data, loading, error, registerAsParticipant]);
+
+  useEffect(() => {
+    if (data) {
+      history.push(`/meeting/${meetingId}`);
```

Comment:
Perhaps add a toast with something like "Du ble lagt til i dette møtet" on the meeting page if a partcipant was redirected from this page.